services/src/main/java/org/keycloak/truststore/TruststoreBuilder.java

```diff
@@ -184,7 +196,7 @@ static KeyStore loadStore(String path, String type, char[] password) {
         }
     }
 
-    private static void mergePemFile(KeyStore truststore, String file, boolean isPem) {
+    private static boolean mergePemFile(KeyStore truststore, String file, boolean isPem, List<String> loadedFiles) {
```

Comment:
`loadedFiles` doesn't seem to be used?

Reply:
It isn't, I had initially added the files in that method, then moved it out for consitency - but didn't remove the parameter. It's gone now.